presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -28,8 +28,7 @@ Java Runtime Environment
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 * 64-bit required
-* version 11 recommended (lower versions not supported, higher versions less
-  tested)
+* version 11.0.7 at minimum (newer major releases are less tested)
```

Comment:
we should remove "less tested" from here, as it can be interpreted they are tested and to the best of my knowledge they are not

Reply:
I used "not tested' in my PR

Reply:
We should required version 11 and mention that patch 11.0.7 or higher is needed

Reply:
I'll change to "not tested" and say "version 12+ not supported"

I don't like saying "version 11" and "patch 11.0.7" since the "patch version" is very specific JDK version terminology that normal users don't know about. They are just going to run `java -version` or go to the download page and see `11.0.9`.

Reply:
I think we can simplify this to

>  version 11.0.7+ (version 12+ not supported)